trustymail/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.2'
```

Comment:
I sense you might be trying to do an actual release; if not I would suggest adding the `-dev` suffix to this.

Additionally, I personally always go with the semantic versioning `X.Y.Z` scheme, so I would change this to `0.2.0`

Reply:
I agree with @IanLee1521 on both points.

Reply:
I agree with @IanLee1521 on both points, especially for first one.